Document how to install stable cli (#2431)
kyma-project/cli: docs/user/README.md

# Kyma CLI

> [!WARNING]

> > All commands in the `alpha` group are still in prototyping stage and its API can slightly change. Use on your own risk.

Reviewer: ```suggestion
> All commands in the `alpha` group are still in the prototyping stage, and their API can change slightly. Use them at your own risk.
```
